cardano-shell.cabal

```diff
@@ -70,6 +70,31 @@ library
                        -Wredundant-constraints
                        -Wpartial-fields
 
+
+
+executable cardano-shell-exe
+  main-is: Main.hs
```

Comment:
What is the purpose of this? I don't get it.

Reply:
It was my impression that this was what was missing for the Coveralls to run. Removed.